sources/packages/backend/apps/api/src/route-controllers/assessment/_tests_/e2e/assessment.students.controller.getAssessmentAwardDetails.e2e-spec.ts

```diff
@@ -0,0 +1,591 @@
+        { msfaaState: MSFAAStates.Signed },
+      ),
+    );
+  });
+
+  it("Should get the student assessment summary containing loan and all grants for a part-time application with a single disbursement.", async () => {
+    // First disbursement values.
+    const firstDisbursementValues = [
+      createFakeDisbursementValue(
+        DisbursementValueType.CanadaLoan,
+        "CSLP",
+        111,
+      ),
+      createFakeDisbursementValue(
+        DisbursementValueType.CanadaGrant,
+        "CSGP",
+        222,
+      ),
+      createFakeDisbursementValue(
+        DisbursementValueType.CanadaGrant,
+        "CSPT",
+        333,
+      ),
+      createFakeDisbursementValue(
+        DisbursementValueType.CanadaGrant,
+        "CSGD",
+        444,
+      ),
+      createFakeDisbursementValue(DisbursementValueType.BCGrant, "BCAG", 555),
+      createFakeDisbursementValue(DisbursementValueType.BCGrant, "SBSD", 666),
+      createFakeDisbursementValue(
+        DisbursementValueType.BCTotalGrant,
+        "BCSG",
+        555 + 666,
+      ),
+    ];
+    // Part-time application with federal and provincial loans and grants.
+    const application = await saveFakeApplicationDisbursements(
+      db.dataSource,
+      {
+        student: sharedStudent,
+        msfaaNumber: sharedMSFAANumber,
+        firstDisbursementValues,
+      },
+      {
+        offeringIntensity: OfferingIntensity.partTime,
+        applicationStatus: ApplicationStatus.Completed,
+      },
+    );
+
+    const [firstSchedule] = application.currentAssessment.disbursementSchedules;
+    await saveFakeDisbursementReceiptsFromDisbursementSchedule(
+      db,
+      firstSchedule,
+    );
+
+    const endpoint = `/students/assessment/${application.currentAssessment.id}/award`;
+    const token = await getStudentToken(
+      FakeStudentUsersTypes.FakeStudentUserType1,
+    );
+
+    // Act/Assert
+    const offering = application.currentAssessment.offering;
+    await request(app.getHttpServer())
+      .get(endpoint)
+      .auth(token, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK)
+      .expect({
+        applicationNumber: application.applicationNumber,
+        applicationStatus: ApplicationStatus.Completed,
+        institutionName: offering.educationProgram.institution.operatingName,
+        offeringIntensity: OfferingIntensity.partTime,
+        offeringStudyStartDate: getDateOnlyFormat(offering.studyStartDate),
+        offeringStudyEndDate: getDateOnlyFormat(offering.studyEndDate),
+        estimatedAward: {
+          disbursement1Date: getDateOnlyFormat(firstSchedule.disbursementDate),
+          disbursement1Status: firstSchedule.disbursementScheduleStatus,
+          disbursement1COEStatus: COEStatus.completed,
+          disbursement1MSFAANumber: "XXXXXXXXXX",
+          disbursement1MSFAAId: sharedMSFAANumber.id,
+          disbursement1MSFAACancelledDate: null,
+          disbursement1MSFAADateSigned: sharedMSFAANumber.dateSigned,
```

Comment:
I believe because of re-using the code from NOA it is returning these values to the API.